### ISSUE TITLE
Show near/medium/far also for the total count

### ISSUE THREAD

**Issue by xandro0777** [marq24/UUID0xFD6FTracer#25]
..thanks

**Comment by marq24** [comment]
I have to admit, that I have difficulties to understand what you would like to see finally on the screen?

Do you want an additional total counters for "near" one for "medium" and another one for "far"? The Challenge I see here is, that a beacon can move from "far" to "near" - what's your expectation in such a situation concerning the "total" value of these fields?

**Comment by xandro0777** [comment]
> Do you want an additional total counters for "near" one for "medium" and another one for "far"? 

exactly.

The Challenge I see here is, that a beacon can move from "far" to "near" - what's your expectation in such a situation concerning the "total" value of these fields?

Should be counted as the nearest it got so I know how many I passed very closely or at some distance.

**Comment by marq24** [comment]
mhh - ok...

Can we agree on the following "total count" behaviour?
1 beacon going from far to near will be counted as total_count_far +1, total_count_medium +1 and total_count_near +1
1 beacon going from medium to far will be counted as total_count_medium +1, total_count_far +1
anything else will add quite a complexity which require additional CPU usage (and the overall goal of the app is to be as battery friendly as possible)...

And have on thing in mind, this near or far count does not replace any real CoronaWarning apps (since there is way more requited ten just detecting beacons)...

Also if you are a bit more interested, you can check out the integrated BTLE Beacon Scanner in my **100 % free** GPSLogger app -> see https://emacberry.com/gpsl/docs/4000-btlescanner/ for details (you might like to play around with the CSV export data).

**Comment by marq24** [comment]
mhh - ok...

Can we agree on the following "total count" behaviour?
1 beacon going from far to near will be counted as total_count_far +1, total_count_medium +1 and total_count_near +1
1 beacon going from medium to far will be counted as total_count_medium +1, total_count_far +1
anything else will add quite a complexity which require additional CPU usage (and the overall goal of the app is to be as battery friendly as possible)...

And have on thing in mind, this near or far count does not replace any real CoronaWarning apps (since there is way more requited ten just detecting beacons)...

Also if you are a bit more interested, you can check out the integrated BTLE Beacon Scanner in my **100 % free** GPSLogger app -> see https://emacberry.com/gpsl/docs/4000-btlescanner/ for details (you might like to play around with the CSV export data).

**Comment by xandro0777** [comment]
Yes, that sounds good enough. Does the app remember beacon it saw previously?

The beacon scanner looks interesting, where do I get it?

**Comment by marq24** [comment]
It's the core idea of the ExposureNotification Beacons that they are not trackable - the beacons change frequently their IDs (so actually it's a complete new beacon) and there is no way to "know" that ID had previously another ID...

In you would like to give GPSLoggers BeaconScanner a try you can get it vie the provided download links here:
https://www.emacberry.com/gpslogger.html

**Comment by xandro0777** [comment]
Don't know for sure, but I thought every beacon lasts 2 weeks? Deduplicating them at least as long they don't change would make a huge difference. Most often I use the app just for a day and reset the counter in the evening anyway.

The GPS Logger is not on F-Droid?

**Comment by marq24** [comment]
> Don't know for sure, but I thought every beacon lasts 2 weeks?

My Experience is different - at least the German CoronaWarnApp Beacons will get every 2-5 minutes a new ID...

> The GPS Logger is not on F-Droid?

Nope - it'S not open sourced (yet) - only Amazon as GooglePlay alternative is available

**Comment by xandro0777** [comment]
Just checked, it is true that the RPI should be exchanged every few minutes.

**Comment by Citro12** [comment]
yes, you can control it with Ramble App

**Comment by marq24** [comment]
> The GPS Logger is not on F-Droid?
Now it is :-)

https://emacberry.com/gpsl/docs/8910-fdroid.html

**Comment by xandro0777** [comment]
Thanks, will try it  very soon :)
